### ISSUE TITLE
When we navigate using TAB key inside the "Available Authorization" dialog keyboard focus moves to background content after the 'Close' button: A11y_VS Code extension for API Center_Definition_OpenAPIDocumentation_Keyboard.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/vscode-azureapicenter#115]
### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG2.4.11; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Keyboard; #Linux; #FTP; #A11yWCAG2.2;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Linux- Ubuntu 22.04.4

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Open API Documentation" using the down arrow and press enter.
7. Demo conference API screen will appear. TAB to "Authorize" button and hit ENTER key.
8. "Available Authorization" dialog will open. Navigate using TAB key and observe the issue.

### Actual Result:
When we navigate using TAB key inside the "Available Authorization" dialog keyboard focus moves to background content after the 'Close' button. Also, with "ESC" key not able to close the dialog.

### Expected Result:
When we navigate using TAB key inside the "Available Authorization" dialog keyboard focus should not move to background content after the 'Close' button. Also, with "ESC" key we should be able to close the dialog.

### User Impact:
Keyboard users will face difficulty here as they will not be able to understand where the focus is and will not be able to interact with the required controls.

### Attachment:

**Comment by AnKushSingh05** [comment]
IRT Bug logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222928

**Comment by Yash14j** [comment]
### Github Tags:
#External;#External:SwaggerUI;#eDAD:3p:222928;#ThirdParty;

**Comment by AnKushSingh05** [comment]
External bug: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222928

**Comment by AnKushSingh05** [comment]
External bug: https://github.com/swagger-api/swagger-ui/issues/10321

**Comment by AnKushSingh05** [comment]
Adding the tag #ResolvedByTT;
